### ISSUE TITLE
New Map Merge Script

### ISSUE THREAD

**Issue by derektan95** [efc-robot/Explore-Bench#10]
Hi @FarawaySail, 

Hope you are well. I am trying out your repository, and it works great. I noticed that you added a `map_merge_new.cpp` within the map_merge folder, along with additional changes to the [original map_merge repository](https://github.com/hrnr/m-explore). 

Would you mind explaining more about the reasons, as well as key considerations, as to why you made these changes? 

Much Thanks, 
Derek

**Comment by FarawaySail** [comment]
When we used the original map_merge repository, we encountered several issues, so we made some corresponding modifications:

1. The original repo only supports merging maps with small resolution, which will crash when the map is large.

2. In the original repo, the coordinate system was not aligned when merging. The origin of the map in ROS is always located in the lower left corner (including the unknown area). When the maps are merged, the origin of the corresponding merged map needs to be modified, but the original repository does not take this into account.
